GencallFormat.py

```diff
@@ -55,6 +56,8 @@ def generate_sample_format_info(self, bpm_records, vcf_record, sample_name):
             sample_name (string): Name of sample
 
         Returns:
-            float: Minimum GenCall score (or None for empty arguments)
+            int: Minimum GenCall score, encoded as a phred quality (or None for empty arguments)
```

Comment:
You will know better but isn't phred score a float?

Reply:
No, its an int